### ISSUE TITLE
Add new taxonomic status value to mark unresolved species/taxa

### ISSUE THREAD

**Issue by mdoering** [CatalogueOfLife/coldp#45]
There is the need in COL to identify taxa, especially species, that have an unresolved taxonomy.
See https://github.com/CatalogueOfLife/data/issues/232 for a case in ITIS.

Any taxon in ColDP can be placed under a parent of any rank. So for example a species can be placed under a family directly avoiding to place it in a certain genus. The problem with bi/trinomials is that they contain an implicit genus (and species) and the name needs to be combined into a new genus (which needs to be done formally published even in botany). This is often not done either because no-one had time to publish this or the new genus to place it under is still unknown. 

This happens quite regularly nowadays with sequence based taxonomic research and is already found in a number of COL sources, in particular ITIS, World Plants and the fungi world.

In order to deal with these taxa a new taxonomic status value `UNRESOLVED` is proposed. This allows:
 - the COL assembly process to avoid creating implicit _split genera_
 - users to list taxa that need further taxonomic work

There is a certain overlap with [provisionally accepted](https://data.catalogueoflife.org/dataset/3LR/names?SECTOR_DATASET_KEY=1141&facet=rank&facet=issue&facet=status&facet=nomStatus&facet=nameType&facet=field&limit=50&offset=0&rank=species&reverse=false&sortBy=taxonomic&status=provisionally%20accepted).

**Comment by mdoering** [comment]
@dhobern @thomasstjerne @gdower @yroskov @olafbanki @DaveNicolson please raise concerns or provide a better name for the actual status.

**Comment by mjy** [comment]
FYI we use a specific relationship/property in ICZN/Botanical names, like this `TaxonNameRelationship::Iczn::Validating::UncertainPlacement`.   It is rank agnostic.

**Comment by DaveNicolson** [comment]
There are also cases where a subspecies (etc., i.e., a trinomial) is not linked to a corresponding species, for the same reason. I would not therefore specifically name it with the word "genus"... We are talking about unresolved_parent or UncertainPlacement (which sounds a lot like _incertae sedis_ to me)....

**Comment by mdoering** [comment]
I agree the GENUS part of the value is wrong. The question is whether we talk about incertae sedis only - isn't the main problem we want to flag that the implicit classification embedded in the name is not accepted? Uncertain placement is broader than that. But maybe still fine? If it's an uncertain placement of a species we know the implicit genus might be wrong.

Possible status names:
 - uncertain placement
 - unresolved classification
 - incertae sedis

**Comment by mdoering** [comment]
`incertae sedis` sounds pretty good to me actually, simple as that

**Comment by mdoering** [comment]
After a small walk in the snow I do wonder if we need any addition at all. Isnt "provisionally accepted" a perfect fit already?

**Comment by DaveNicolson** [comment]
Do you want all "provisionally accepted" names to be treated like this too?

**Comment by DaveNicolson** [comment]
And in some cases, the names are definitely accepted, it is just a question of where they belong. I don't think this fits "provisionally accepted"...

**Comment by gdower** [comment]
I would not use `incertae sedis` as not all CoL users will know what that means. Some users have mistakenly taken `incertae sedis` as a species name in the past. It might be better to just use `uncertain placement`.

**Comment by mdoering** [comment]
Does "uncertain placement" always imply the name is accepted? I very much like that we can clearly identify from all status values so far whether they are accepted or synonyms. For `provisionally accepted` we have:

> Treated as accepted, but doubtful whether this is correct.

The important part for an uncertain placement is the actual parent classification. I am not yet convinced uncertain placement and provisionally accepted differ that much.

**Comment by dhobern** [comment]
Clearly in a sense, all of the cases relate to uncertain taxonomic placement for the organisms in question, and in many cases this will be in part because no obviously suitable taxon has been described at a higher level to contain them.

However, I do think we need two flags, or at least a special case flag. In many cases, no one is really querying the assignment of a species to a genus - the question is where that genus should be classified. This is no different from problems with higher classification. These could all be covered by "uncertain placement"/"incertae sedis".

The special case of species currently in an otherwise accepted genus where taxonomists believe they should not be in that genus (but where no destination genus yet exists) should certainly be noted as an issue of "uncertain placement"/"incertae sedis" BUT it should also be flagged and noted that the binomial combination is known to be wrong but cannot yet be fixed. Knowing that this is the case is a very powerful thing. Look in iNaturalist. There are records for Dichelia clarana, a tortricid moth, but Dichelia is a western Palearctic genus and this Australian species is only superficially similar. The Australian Faunal Directory has the species placed in a loose "Clarana group" and then has bizarrely constructed the binomial Clarana clarana for it. This implies a described genus Clarana and that the correct binomial is Clarana clarana. iNaturalist accordingly has many other records assigned to this name. Neither Dichelia clarana nor Clarana clarana is "correct", but the only name given under the code is Dichelia clarana and we want to make clear that the species is nevertheless NOT in Dichelia. Simply flagging it as incertae sedis would leave many people assuming that the binomial is correct. I know that maybe we could rely on the species here being incertae sedis and in other cases (where the binomial is accepted) rely on the genus being incertae sedis and the species not being marked this way, but I would really like a more explicit flag in these cases so they can be tracked and we can use the information to format the binomials in a consistent way that shows the uncertainty (parentheses, quotation marks or whatever else).

**Comment by jliljeblad** [comment]
There is a similar case that we have a flag for in the Swedish taxonomic database Dyntaxa. It is not uncommon that a revision is published where a genus is synonymized with another and a bunch of species are moved to the new genus and, hence, appear in a new or re-established combination. Often, however, there are more species in the synonymized genus that are not dealt with in the revision. If we adopt the genus synonymization, we are forced to move also those species which are not dealt with. We cannot, especially with the ICN, not do the new combinations ourselves without formal publication. In this case we have a flag that says "Combination does not match parent genus". Now, we can explain why species with the wrong genus combination are in this genus. Otherwise we will get people telling us we have made a mistake. The alternative, I guess, is to place the problematic species in a "Genus, incertae sedis", but that would more of an active choice on our side. I think moving them over to the new genus is more correct, since that is what the old genus synonymized into.

Maybe a single flag could cover both this and Donald's case although the combinations are wrong for different reasons.

**Comment by mdoering** [comment]
Fully agree that the main problem we should highlight is the incorrect genus (or maybe even species?) as part of the binomial, not so much the uncertain placement. The fact that we are uncertain where to place it does not need much explicit attention I believe. The current practice to link it to a higher taxon, skipping a genus or, as in @jliljeblad example, place it within a new genus while keeping the old combination with the "wrong" genus works just fine. 

The COL importer actually flags a parent mismatch if the implicit genus of the species does not match its parent genus.

**Comment by mdoering** [comment]
It seems a new status to flag the uncertain or wrong genus placement of binomials is needed. As this status is very specific for a wrong genus placement only due to the nature of binomials it could be `UNRESOLVED_GENUS ` or `UNCERTAIN_GENUS`. But that does not make clear it is an _accepted_  species status. We can obviously document this, but maybe some status with ACCEPTED in its name would be better. `ACCEPTED_BUT_UNCERTAIN_GENUS` is awfully long though.

**Comment by yroskov** [comment]
Status "Provisionally Accepted" with genus name meets the expectation.

**Comment by yroskov** [comment]
Not-established genus might be present in the export file as: 
(a) original genus name in Genus field (that creates "split" species sets between two different locations in the classification), as in ITIS,
(b) modified original genus name (e.g. [Lagostomus], or Valdiviana?),
(c) any artificial name (*GAgromyzidae, or Neanuratobeassignedtopropergenus, GenusnovnotDicranocentrus), 
(d) empty Genus field

Are we going to to unify presentation of not-established taxa in CoL?

**Comment by mdoering** [comment]
> Status "**Provisionally Accepted**" with genus name meets the expectation.

We need to be able to identify species with a unresolved genus, e.g. that the COL syncs do not generate those bad genera but instead keep the species directly under the family or whatever the next higher taxon would be. That should be achievable with status=provisional and linking the species to the family, skipping any genus record. I was wondering about the use of _Provisionally Accepted_ to flag these cases, avoiding the need for an extra status. We could recommend to use the taxon remarks field for such cases and populate that with a note that explains the situation. It might be worth recommending to provide a taxon remark for all provisional taxa explaining the reason behind it.

In case providers supply c) that should be considered bad data and prevented as much as possible. Or otherwise edited out of COL manually.
